src/Microsoft.Extensions.Testing.Abstractions/project.json

```diff
@@ -12,7 +12,8 @@
       "target": "project"
     },
     "Microsoft.DiaSymReader": "1.0.8",
-    "Microsoft.DiaSymReader.Native": "1.4.0-rc2"
+    "Microsoft.DiaSymReader.Native": "1.4.0-rc2",
+    "System.Reflection.Metadata": "1.4.1-beta-24410-02"
```

Comment:
The reason why I added it here is because TestingAbstractions itself depends on Reflection.Metadata and before it was relying on transitive dependency to bring it in. I believe that is wrong and that we should be explicit about compile time dependencies. It is ok to depend on the code in ProjectModel, I don't believe it is ok to depend on ProjectModel's dependencies.